README.md

```diff
@@ -0,0 +1,191 @@
+# Serverless Log
+
+This is some experimental tooling which allows for the maintenance and querying
+of a log represented by tiles & files. These tools are built upon components
+from the Trillian repo (e.g. compact ranges), but *do not* use the Trillian Log
+service itself.
```

Comment:
Is the second sentence here still true? Didn't compact ranges get moved to a separate repo? If so, maybe it's more clear to not mention Trillian here at all.

Reply:
Also ok with doing this in a separate PR.